src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,719 @@
+      example: True
+    expense_include_foreign_currency:
+      type: boolean
+      description: |
+        Should policy run on expenses created in foreign currency.
+      example: True
+    expense_flight_travel_class:
+      $ref: ./fields.yaml#/expense_policy_flight_travel_classes
+    expense_train_travel_class:
+      type: array
+      description: >
+        This field is applicable only for Train category.
+
+        Index 0 holds onward journey travel class, index 1 holds return journey
+        travel class.
+      items:
+        type: string
+      example:
+        - AC First Class
+        - Second Sitting
+    expense_fiscal_year_start_month:
+      type: number
+      description: | 
+        Month number of start of fiscal year.
+      example: 4
+      minimum: 1
+      maximum: 12
+    expense_vehicle_type:
+      allOf:
+        - $ref: ./fields.yaml#/vehicle_type
+      description: |
+        Vehicle type targeted by this policy.
+    expense_annual_distance_max:
+      type: number
+      example: 1200
+      description: |
+        Maximum travel distance allowed, above which policy will be violated.
+    employee_level_ids:
+      type: array
+      description: |
+        Employee level ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - lvl1sdfad
+        - lvl2sdfas
+    employee_level_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_custom_fields:
+      allOf:
+        - $ref: './fields.yaml#/custom_fields'
+      description: |
+        Custom fields
+    employee_custom_fields_op:
+      type: object
+      description: |
+        Operator for the employee_custom_fields.
+      properties:
+        field_name:
+          type: string
+    employee_department_ids:
+      type: array
+      description: |
+        Employee department ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - dept1asdfa
+        - dept2adfad
+    employee_department_id_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_locations:
+      description: |
+        Location of employees targeted by this policy.
+      type: array
+      items:
+        $ref: './fields.yaml#/location'
+    employee_locations_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_user_ids:
+      type: array
+      description: |
+        Employee user ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - usabcds1
+        - usabcds2
+    employee_user_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_titles:
+      type: array
+      description: |
+        Employee titles targeted by this policy.
+      items:
+        type: string
+      example:
+        - CEO
+        - Manager
+    employee_titles_op:
+      $ref: ./fields.yaml#/in_operator
+    action_flag:
+      type: boolean
+      description: |
+        Whether to flag the expense when it violates policy.
```

Comment:
- Whether to flag the expense when it violates this policy rule.
- If it is set to true, then the expense(s) violating this policy rule, will be flagged.


just check once, do we use any `default` keyword for default values of fields - applies to every field